lib/ui/window/platform_configuration.cc

```diff
@@ -589,6 +589,16 @@ void PlatformConfigurationNativeApi::ScheduleFrame() {
   UIDartState::Current()->platform_configuration()->client()->ScheduleFrame();
 }
 
+void PlatformConfigurationNativeApi::ForceSyncFrame() {
+  // TODO(dkwingsmt): This method is not implemented and is not used for now.
+  // This is because we have to land the Dart FFI method first before being able
+  // to run the performance test for the full changes due to the restriction of
+  // the performance test. Eventually, we should either land the full changes,
+  // or remove this method.
+  // https://github.com/flutter/flutter/issues/142851
+  FML_UNREACHABLE();
```

Comment:
This is in fact quite reachable. You could error log if you want to discourage calling this method but leaving a process exit accessible from dart:ui does not seem reasonable.

Reply:
This PR has been merged to https://github.com/flutter/engine/pull/49950 because I realized that this PR does not improve the situation alone. Now this method is implemented.